core/src/main/resources/db/migration/V2__add_pkg_sigs.sql

```diff
@@ -16,7 +16,10 @@ CREATE TABLE OperationResult (
     update_request_id CHAR(36) NOT NULL,
     result_code INT NOT NULL,
     result_text TEXT NOT NULL,
+    vin varchar(64) NOT NULL,
 
     PRIMARY KEY (id),
-    FOREIGN KEY fk_operation_result_update_request_id (update_request_id) REFERENCES UpdateRequest(update_request_id)
+    FOREIGN KEY fk_operation_result_update_request_id (update_request_id) REFERENCES UpdateRequest(update_request_id),
+    FOREIGN KEY fk_operation_result_vin (vin) REFERENCES Vehicle(vin)
 );
+
```

Comment:
Changing these files will break migrations, as the checksum will be different and we will not be able to run flywayMigrate in mesos or locally. If we need to change the schema we need to add new migrations.